static/app/views/performance/trends/changedTransactions.tsx

```diff
@@ -575,6 +613,7 @@ function TransactionSummaryLink(props: TransactionSummaryLinkProps) {
     projects,
     location,
     currentTrendFunction,
+    setState,
```

Comment:
Let's not pass down a generic `setState` since it's not clear what this prop is, instead let's make it a `onItemClicked` and bind it to `setState`.